SConstruct

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-EnsureSConsVersion(3, 0, 0)
+EnsureSConsVersion(4, 2, 0)
 EnsurePythonVersion(3, 5)
```

Comment:
Not for this PR specifically, but a note to self: SCons 4.2.0 is deprecated Python 3.5 support, so we might want to follow suit and get ready for 4.3.0 by bumping this to 3.6 (and then start porting the codebase to finally benefit from f-strings).

Reply:
done in 
e3b3d25